src/sinks/prometheus/remote_write.rs

```diff
@@ -1,4 +1,5 @@
 use super::collector::{self, MetricCollector as _};
+use crate::http::Auth;
```

Comment:
Since we already have a block for all the `crate` imports, please move this into that block.